icu4c/source/test/intltest/unitstest.cpp

```diff
@@ -52,6 +56,75 @@ void UnitsTest::runIndexedTest(int32_t index, UBool exec, const char *&name, cha
     TESTCASE_AUTO_END;
 }
 
+// Tests freshness of the constants in units.txt by simply loading each directly
+// specified conversion rate into UnitConverter.
+void UnitsTest::testConstantFreshness() {
```

Comment:
What are you testing?  There are no assertions in this function.

Reply:
Example output when I change the "acre to meter-squared" definition to use a non-existent "acore_to_m2" constant:

         testConstantFreshness data: expected success but got error: U_INVALID_FORMAT_ERROR - UnitConverter(<acre>, <square-meter>, status).
         If U_INVALID_FORMAT_ERROR, please check that "icu4c/source/i18n/unitconverter.cpp" has all constants?
         Factor for "acre" is: "acre_to_m2".
         Full list of constants: "G" "PI" "ft2_to_m2" "ft3_to_m3" "ft_to_m" "gal_imp_to_m3" "gal_to_m3" "gravity" "in3_to_m3" "lb_to_kg" - (Are you missing data?)